src/Controller/PetController.php

```diff
@@ -31,7 +31,9 @@ public function __construct(PetService $petService)
     public function new(Request $request) : Response
     {
         // creates a pet and creates a form for adding new pet
-        $form = $this->createForm(PetForm::class, $pet = new Pet());
+        $form = $this->createForm(PetForm::class, $pet = new Pet(), [
+                'validation_groups' => ['Default', 'registration']
```

Comment:
why registration name of group? this is pet

Reply:
because it should be evaluated just when we register new pet

Reply:
ah, if you say it like that, should then the action and route have `register` instead of new/create